server/admin/webserver/api.go

```diff
@@ -0,0 +1,33 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package webserver
```

Comment:
Do we envision other packages under `server/admin`, like non-http options or just refactoring into different packages?  It seems like this is going to be the admin package in its entirety right now.